tests/refinements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2466,8 +2465,7 @@ type Breakfast = Apple | Orange | Broccoli | Carrot;
 function bar(x: Breakfast) {
   if (x.kind === \"Fruit\") {
     (x.taste: \"Good\");
-  } // error, Apple.taste = Bad else
-    (x.raw: \"No\"); // error, Carrot.raw = Maybe
+  } // error, Apple.taste = Bad else (x.raw: \"No\"); // error, Carrot.raw = Maybe
```

Comment:
Seems like that invalid code was already there before I changed that :| Maybe I'll have some time to look through the comments code soon.